### PR TITLE
Modify tags api index to allow for bulk GET of tags

### PR DESCRIPTION
**Pull request by dscottS3**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Modify tags index action to accept `tags_id[]` query param to search for tags.

## Related Tickets & Documents

#16162 

## QA Instructions, Screenshots, Recordings

Make a GET request to `locale/en/api/tags.json` with query param `tag_ids` as an array. Example:

`locale/en/api/tags.json?tag_ids[]=2&tag_ids[]=10`

### UI accessibility concerns?

n/a

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

**Comment by jeremyf** [comment]
@aitchiss could you look to see if this will meet your needs.

File: app/controllers/api/v0/tags_controller.rb [hunk 1]
```diff
@@ -3,17 +3,19 @@ module V0
     class TagsController < ApiController
       before_action :set_cache_control_headers, only: %i[index]
 
-      ATTRIBUTES_FOR_SERIALIZATION = %i[id name bg_color_hex text_color_hex].freeze
+      ATTRIBUTES_FOR_SERIALIZATION = %i[id name bg_color_hex text_color_hex short_summary].freeze
```
Comment (aitchiss) [inline app/controllers/api/v0/tags_controller.rb:6]: We'll also need the badge image url in the payload - e.g. 

```
 "badge": { 
        "badge_image": {
            "url":"/uploads/badge/badge_image/1/8.png"
        }
    }
```

File: spec/requests/api/v0/tags_spec.rb [hunk 2]
```diff
@@ -16,11 +16,12 @@
       get api_tags_path
 
       response_tag = response.parsed_body.first
-      expect(response_tag.keys).to match_array(%w[id name bg_color_hex text_color_hex])
+      expect(response_tag.keys).to match_array(%w[id name bg_color_hex text_color_hex short_summary badge])
       expect(response_tag["id"]).to eq(tag.id)
       expect(response_tag["name"]).to eq(tag.name)
       expect(response_tag["bg_color_hex"]).to eq(tag.bg_color_hex)
       expect(response_tag["text_color_hex"]).to eq(tag.text_color_hex)
+      expect(response_tag["short_summary"]).to eq(tag.short_summary)
```
Comment (jeremyf) [inline spec/requests/api/v0/tags_spec.rb:24]: What does the "badge" value look like?  That appears untested.

Comment (dscottS3) [reply spec/requests/api/v0/tags_spec.rb]: updated!

**Review by jeremyf — APPROVED** [review]

**Review by atsmith813 — APPROVED** [review]

**Pull request merged**